src/Notifications/Notifications.csproj

```diff
@@ -8,7 +8,7 @@
   <PropertyGroup Condition=" '$(RunConfiguration)' == 'Notifications-SelfHost' " />
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="6.0.4" />
```

Comment:
⛏️ I would bump this at the same time.

Reply:
I disagree for something that we want to pick this late into testing, they don't have dependencies on each other and they both interact with separate parts of the SignalR stack. I'm fine with making a separate PR that only goes into `master` though. Would that be okay?